ISSUE_TEMPLATE

```diff
@@ -0,0 +1,6 @@
+<!-- Hello! If you're filing a bug, please include every step so as to help us reproduce it on our machines. If you're unsure about how to file an issue, use the issue template. If you need any help regarding usage of coala, check out the documentation or hit us up on the chat-room. You can ignore or delete this text, it is commented and won't appear when the issue is submitted or previewed.
```

Comment:
I have done it already :)  If you find any specific point which I missed then please let me know